python/paddle/__init__.py

```diff
@@ -811,6 +812,7 @@
     'unique_consecutive',
     'set_cuda_rng_state',
     'set_rng_state',
+    'register_rng_state_as_index',
```

Comment:
`register_rng_state_as_index` should be public API?

Reply:
move these APIs to incubate